server/templates/place.html

```diff
@@ -17,21 +17,25 @@
 
 {% set main_id = 'dc-places' %}
 {% set page_id = 'page-dc-places' %}
-{% set title = place_name + ' ' + category + ' - ' + _('Place Explorer') %}
+{% if category == '' %}
+  {% set title = place_name + ' - ' + _('Place Explorer') %}
+  {% set description = _('Statistics about economics, health, equity, crime, education, demographics, housing, and environment in {place_name}.'.format(place_name=place_name)) %}
+{% else %}
+  {% set title = place_name + ' - '+ _('CHART_TITLE-CHART_CATEGORY-{category}'.format(category=category)) + ' - ' + _('Place Explorer') %}
+  {% set description = _('Statistics about {category} in {place_name}.'.format(category=category, place_name=place_name)) %}
+{% endif %}
 {% set subpage_title = _('Place Explorer') %}
 {% set subpage_url = url_for('place.place') %}
 {% set place_category = category %}
-{% set canonical_base_url = url_for('place.place', place_dcid=place_dcid) %}
-{% set canonical_category_url = url_for('place.place', place_dcid=place_dcid, category=category) %}
 
 {% block head %}
 <link rel="stylesheet" href={{url_for('static', filename='css/place_page.min.css' , t=config['VERSION'])}}>
 <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" />
-{% if category == '' %}
-<link rel="canonical" href="https://{{ request.host }}{{ canonical_base_url }}" />
-{% elif category != '' %}
-<link rel="canonical" href="https://{{ request.host }}{{ canonical_category_url }}" />
+
+{% if locale == "en" %}
```

Comment:
do we still need this check?

Reply:
Yes, right now we don't have translations for the descriptions, and getting those translations will take some time. The description string defaults to the original text when a translation is not available, so we'd end up with english descriptions for non-english pages in the meantime.

Reply:
That makes sense- thank you!